akka-docs/src/main/paradox/typed/cluster-membership.md

```diff
@@ -115,35 +115,29 @@ members in quorum decisions.
 
 ## State Diagrams
 
-### State Diagram for the Member States (`akka.cluster.allow-weakly-up-members=off`)
+### State Diagram for the Member States 
 
-![member-states.png](../images/member-states.png)
-
-### State Diagram for the Member States (`akka.cluster.allow-weakly-up-members=on`)
-
-![member-states-weakly-up.png](../images/member-states-weakly-up.png)
+![member-states.png](../images/member-state-diagram.png)
   
 #### User Actions
 
  * **join** - join a single node to a cluster - can be explicit or automatic on
 startup if a node to join have been specified in the configuration
    
- * **leave** - tell a node to leave the cluster gracefully
+ * **leave** - tell a node to leave the cluster gracefully, normally triggered by ActorSystem or JVM shutdown through @ref[coordinated shutdown](../coordinated-shutdown.md)
    
- * **down** - mark a node as down
+ * **down** - mark a node as down - action required to remove crashed nodes from the cluster, can be triggered manually, through [Cluster HTTP Management](https://doc.akka.io/docs/akka-management/current/cluster-http-management.html#put-cluster-members-address-responses), or it is triggered automatically by a @ref[downing provider](cluster.md#downing) like [Split Brain Resolver](https://doc.akka.io/docs/akka-enhancements/current/split-brain-resolver.html)
```

Comment:
```suggestion
 * **down** - mark a node as down. This action is required to remove crashed nodes (that did not 'leave') from the cluster. It can be triggered manually, through [Cluster HTTP Management](https://doc.akka.io/docs/akka-management/current/cluster-http-management.html#put-cluster-members-address-responses), or automatically by a @ref[downing provider](cluster.md#downing) like [Split Brain Resolver](https://doc.akka.io/docs/akka-enhancements/current/split-brain-resolver.html)
```